pkg/test/framework/image/flags.go

```diff
@@ -58,6 +58,9 @@ func init() {
 		"Common Container tag to use when deploying container images")
 	flag.StringVar(&settingsFromCommandLine.PullPolicy, "istio.test.pullpolicy", settingsFromCommandLine.PullPolicy,
 		"Common image pull policy to use when deploying container images")
+	flag.StringVar(&settingsFromCommandLine.ImagePullSecret, "istio.test.imagePullSecret", settingsFromCommandLine.ImagePullSecret,
+		"Secret that already exists in cluster(s) to use for pulling images in test app deployments. "+
```

Comment:
To me, the description heavily implies its taking the secret name here, rather than a Secret yaml. Can you clarify the comment?

Reply:
Ah right.. I changed direction halfway through implementing to use the file.